PyRDF/backend/Dist.py

```diff
@@ -175,9 +175,8 @@ def reducer(values_list1, values_list2):
                 if isinstance(values_list1[i], ROOT.TH1) or isinstance(values_list1[i], ROOT.TH2):
                     # Merging two objects of type ROOT.TH1D or ROOT.TH2D
                     values_list1[i].Add(values_list2[i])
-                elif isinstance(values_list1[i], int):
-                    # Merging two objects of type `int`
-                    values_list1[i] += values_list2[i]
+                else:
+                    raise NotImplementedError("The reducer function doesn't support operations that give a integer output yet !")
```

Comment:
The `else` here doesn't just cover integers, I would just say that `type(values_list1[i]` cannot be merged.
On the other hand, we need to be able to merge  more things here, as discussed in the previous meeting. `TProfile`s are covered by `TH1` (it is a subclass), but we still need `TGraph`. Can you add a check here for the `TGraph` type? In order to merge, you can use:
https://root.cern.ch/doc/v614/classTGraph.html#a10c31c206c4559803156b9b4a4b8b997